test/thermodynamics/first_law_of_termodynamics_test.py

```diff
@@ -0,0 +1,41 @@
+from collections import namedtuple
+from pytest import approx, fixture, raises
+from symplyphysics import (
+    errors,
+    units,
+    Quantity,
+    SI,
+    convert_to,
+)
+from symplyphysics.laws.thermodynamics import first_law_of_termodynamics as first_law_of_term
+
+
+@fixture(name='test_args')
+def fixture_test_args():
+    W = Quantity(50 * 10 ** 3 * units.joule)
```

Comment:
Do you really think that `50 * 10 ** 3` looks better than `50000`?
BTW we support units with prefixes, eg `units.kilo`